src/main/java/org/kainos/ea/cli/SalesEmployee.java

```diff
@@ -0,0 +1,28 @@
+package org.kainos.ea.cli;
+
+public class SalesEmployee extends Employee{
+    public SalesEmployee(int employeeID, double salary, String bankAccountNumber, String NINumber, float comissionRate, int salesEmployeeID) {
+        super(employeeID, salary, bankAccountNumber, NINumber);
+        setComissionRate(comissionRate);
+        setSalesEmployeeID(salesEmployeeID);
+    }
+    private float comissionRate;
+
+    private int salesEmployeeID;
+
+    public int getSalesEmployeeID() {
+        return salesEmployeeID;
+    }
```

Comment:
do you need a delivery employee ID? if you use this in the db it will prevent you from using the key integrity to only allow an EmployeeID to exist in the DeliveryEmployee table multiple times